src/fprime_gds/executables/run_deployment.py

```diff
@@ -174,6 +209,20 @@ def main():
     # Launch the desired GUI package
     launchers.append(launch_html)
 
+    # Check if OpenMCT is set to be used. If true, generate OpenMCT States and Launch the OpenMCT Server
+    if parsed_args.openmct:
+
```

Comment:
... e.g. 
```
if OPENMCT_NOT_INSTALLED: 
     [raise "Error: OpenMCT not installed. Install with....]
```
Otherwise it'll just raise weird errors when trying to use the symbols below.